config/eye.yml.erb

```diff
@@ -58,7 +56,8 @@ processes:
       user_commands:
         rotate: "kill -USR1 {PID}"
 
-<% if is_first_host %>
+# This should only run on 1 instance.
+# See run_auto_cancel in secrets.yml
```

Comment:
```suggestion
  # This should only run on 1 instance.
  # See run_auto_cancel in secrets.yml
```

Reply:
This is valid, but I think it should be indented